app/src/main/resources/database/scripts/initial-schema.sql

```diff
@@ -6,4 +6,12 @@ CREATE TABLE recipe (
 	target_vg smallint NOT NULL,
 	base_nicotine smallint NOT NULL,
 	quantity smallint
+);
+
+CREATE TABLE recipe_flavourings (
+    recipe_id serial UNIQUE NOT NULL,
```

Comment:
The ```recipe_id``` cannot be serial UNIQUE in this table, because serial is used for autogenerated ids, this id is not going to be generated, it's going to be referenced on the table recipe. 
Use only: ```recipe_id bigint not null``` instead.